front/lib/api/assistant/conversation.ts

```diff
@@ -300,6 +316,9 @@ export async function getUserConversations(
         model: Conversation,
         as: "conversation",
         required: true,
+        where: {
+          ...(includeDeleted ? {} : { visibility: { [Op.ne]: "deleted" } }),
```

Comment:
I fear this query will be super slow? Did you benchmark it? 

Lower risk diff is to filter in the reduce below. But if you're 100% sure it's fast (it has because it's run often and it's a bit hungry) then fine with this.

Reply:
Did not benchmark; 90% sure that it will get the ConversationParticipants, then the conversation ids that match then use the where

90% != 100% => using filter

Reply:
I've had super bad surprises in the past with join peformance through sequelize :grimacing: